apidoc/Titanium/UI/WebView.yml

```diff
@@ -393,8 +393,8 @@ events:
     summary: Fired when a blacklisted Url is stopped
     description: |
         This event is fired when a blacklisted Url is stopped
-    platforms: [android]
-    since: "5.4.0"
+    platforms: [android, iphone, ipad]
+    since: [android: "5.4.0", iphone: "6.1.0", ipad: "6.1.0"]
```

Comment:
Use curly brackets (like an object) instead of square brackets. Same for the below as well